product-catalog/app/productcatalog/services/ProductProjectionService.java

```diff
@@ -0,0 +1,74 @@
+package productcatalog.services;
+
+import io.sphere.sdk.categories.Category;
+import io.sphere.sdk.client.PlayJavaSphereClient;
+import io.sphere.sdk.products.ProductProjection;
+import io.sphere.sdk.products.ProductVariant;
+import io.sphere.sdk.products.queries.ProductProjectionQuery;
+import io.sphere.sdk.products.search.ProductProjectionSearch;
+import io.sphere.sdk.queries.PagedQueryResult;
+import io.sphere.sdk.queries.PagedResult;
+import play.libs.F;
+
+import javax.inject.Inject;
+import java.util.*;
+
+public class ProductProjectionService {
```

Comment:
Normally we split every Service in an interface and an Impl class. The interface contains verbose Javadoc what it does.